tasks/AccelerationController.cpp

```diff
@@ -31,10 +31,35 @@ bool AccelerationController::configureHook()
     cmdVector = Eigen::VectorXd::Zero(numberOfThrusters);
     expectedEffortVector = Eigen::VectorXd::Zero(6);
     names = _names.get();
+    base::MatrixXd weighingMatrix;
+    base::VectorXd thrustersWeights = _thrusters_weights.get();
+
+    if(thrustersWeights.size() != numberOfThrusters)
+        return false;
```

Comment:
Which one do you suggest using for log messages: base/Logging or std::runtime_error